crawler.py

```diff
@@ -87,15 +89,26 @@ def test_crash(driver):
 var crash = badptr.contents;""")
 
 
-def get_domain_list(n_sites):
+def get_domain_list(n_sites, exclude_option):
     """Get the top n sites from the tranco list"""
-    tr = Tranco(cache=False)
-
-    if not n_sites:
-        n_sites = DEFAULT_NUM_SITES
-
-    return tr.list().top(n_sites)
-
+    tr = Tranco(cache=False).list().top()
```

Comment:
Nit: `tr` isn't the best name anymore since this is no longer a Tranco object, but a list of domains.